### ISSUE TITLE
y_pred, y_true shape, are different: loss.py:535: UserWarning: Using a target size (torch.Size([1, 1])) that is different to the  input size (torch.Size([10, 1])).

### ISSUE THREAD

**Issue by mw66** [andrewgcodes/xlstm#4]
site-packages/torch/nn/modules/loss.py:535: UserWarning: Using a target size (torch.Size([1, 1])) that is different to the 
input size (torch.Size([10, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.                                                       
  return F.mse_loss(input, target, reduction=self.reduction)

If we set break point, and print out the y_pred, y_true shape, you can see they are different:

```
> /test/python/mLSTM.py(87)<module>()
-> loss += criterion(y_pred, y_true)
(Pdb) p y_pred.shape
torch.Size([10, 1])
(Pdb) p y_true.shape
torch.Size([1, 1])
(Pdb) 
```

**Comment by mw66** [comment]
I think the line need to be fixed as:
```
        loss += criterion(y_pred[0], y_true[0])                                                                                                                                                
```
